test.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash')
+const usbboot = require('./lib/shared/sdk/usbboot')
+const usb = require('usb')
+const fs = require('fs')
+
+usbboot.scan().then((devices) => {
+  console.log(devices[0].instance)
+  return usbboot.flash(_.first(devices), {
+    files: {
+      'bootcode.bin': fs.readFileSync('../usbboot/msd/bootcode.bin'),
```

Comment:
Maybe we need to add a script that gets run during Etcher's build process, to download the latest version of these files from https://github.com/raspberrypi/usbboot ? (or maybe we should version-lock to a specific commit hash?)
If we include these files within Etcher itself, I wonder if we need to update our LICENSE file? \*shrug\*